### PR TITLE
Add new orderer to channel 

### LINKED ISSUE #70: Add support to add a new orderer

### PR DESCRIPTION
**Pull request by sapthasurendran**
closes #70

### CODE REVIEW

File: fabric-network/docker-based-syschannel/organizations/cryptogen/crypto-config-orderer-new.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  # Orderer
+  # ---------------------------------------------------------------------------
+  - Name: Orderer
+    Domain: example.com
+    EnableNodeOUs: true
+    # ---------------------------------------------------------------------------
+    # "Specs" - See PeerOrgs for complete description
+    # ---------------------------------------------------------------------------
+    Specs:
+      - Hostname: orderer
```
Comment (Param-S) [inline fabric-network/docker-based-syschannel/organizations/cryptogen/crypto-config-orderer-new.yaml:20]: if we want to generate the crypto only for orderer6, then we can remove this entry `Hostname: orderer`

File: fabric-network/docker-based-syschannel/scripts/addNewOrderer.sh [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+#
+# Copyright IBM Corp. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# import utils
+. scripts/envVar.sh
+
+# fetchChannelConfig <org> <channel_id> <output_json>
```
Comment (Param-S) [inline fabric-network/docker-based-syschannel/scripts/addNewOrderer.sh:11]: I think this comment needs to go down closer to function definition.

**Comment by sapthasurendran** [comment]
@davidkel  I have updated the scripts.  Works fine.

**Review by davidkel — APPROVED** [review]
LGTM

**Pull request merged**